pcp/Instance.c

```diff
@@ -0,0 +1,160 @@
+/*
+htop - Instance.c
+(C) 2022-2023 Sohaib Mohammed
+(C) 2022-2023 htop dev team
```

Comment:
Please check consistency/order of copyright headers for the newly created PCP platform files.

Also the filename above should include the pcp/ path prefix, as present with other files.

Reply:
Ack, thanks.